features/add_dishes.feature

```diff
@@ -1,24 +1,35 @@
-Feature: Add dishes to menu
-  As a user
-  In order to decide from various dishes from a restaurant
-  I would like to see many dishes in the restaurant´s menu
+Feature: Add dishes to restaurant
+  As a restaurant owner
+  In order to manage my menu
+  I would like to be able to add dishes on my menu
```

Comment:
We discussed with @casperjureskog earlier this morning and we realised it was grammatical oversight on there side. The editing of this section was done together with him. Git blame is blaming me and I dont remember commiting this file :thinking: